test/gurka/CMakeLists.txt

```diff
@@ -1,6 +1,10 @@
 if(ENABLE_DATA_TOOLS)
   file(GLOB_RECURSE TEST_FILES "${CMAKE_CURRENT_LIST_DIR}/test_*.cc")
 
+  add_library(valhalla_gurka_test STATIC gurka.h gurka.cc)
+  target_include_directories(valhalla_gurka_test PUBLIC ${VALHALLA_SOURCE_DIR}/test/gurka)
+  target_link_libraries(valhalla_gurka_test valhalla_test valhalla_datatools_test)
```

Comment:
wont we have to also link this new target over in test/CMakeLists.txt for any of the data tools unit tests? Maybe none of them are using stuff from here yet but I suspect we should do it anyway so someone isnt hit by a gotcha later on

Reply:
Yes, you're right.  I will do this